crates/zed/src/languages/swift/highlights.scm

```diff
@@ -0,0 +1,290 @@
+[
+  "."
+  ";"
+  ":"
+  ","
+] @punctuation.delimiter
+
+; TODO: "\\(" ")" in interpolations should be @punctuation.special
+[
+  "\\("
+  "("
+  ")"
+  "["
+  "]"
+  "{"
+  "}"
+] @punctuation.bracket
+
+; Identifiers
+(attribute) @variable
+
+(type_identifier) @type
+
+(self_expression) @variable.builtin
+
+; Declarations
+"func" @keyword.function
+
+[
+  (visibility_modifier)
+  (member_modifier)
+  (function_modifier)
+  (property_modifier)
+  (parameter_modifier)
+  (inheritance_modifier)
+  (mutation_modifier)
+] @type.qualifier
+
+(function_declaration
+  (simple_identifier) @function.method)
+
+(function_declaration
+  "init" @constructor)
+
+(throws) @keyword
+
+(where_keyword) @keyword
+
+(parameter
+  external_name: (simple_identifier) @variable.parameter)
```

Comment:
We don't have a `@variable.parameter`, you can see what we do have in [syntax.rs](https://github.com/zed-industries/zed/blob/main/crates/theme/src/styles/syntax.rs#L16-L68)